grpc/src/main/go/proto/language_id.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+option java_package = "zemberek.proto";
+option java_multiple_files = true;
+
+package zemberek.langid;
+
+message LanguageIdRequest {
+  string input = 1;
+  int32 maxSampleCount = 2;
+}
+
+message LanguageIdResponse {
+  string langId = 1;
+}
+
+message IdResult {
+  string id = 1;
+  double score = 2;
+}
+
+message LanguageIdScoresResponse{
+  repeated IdResult idResult = 1;
+}
+
+service LanguageIdService {
```

Comment:
Why is this proto different than the one in grpc/src/main/proto/language_id.proto ? Also it should be preferrabe to have single proto file instead of duplication

Reply:
removed.